app/models/social-link.js

```diff
@@ -17,7 +17,13 @@ export default ModelBase.extend({
     // and non-null name is being sent from API, for some reason, for certain events,
     // this throws an error, so we check first if name exists
     // https://github.com/fossasia/open-event-frontend/issues/4777
-    return this.name && this.name.trim().toLowerCase();
+
+    let normalizedName = this.name.trim().toLowerCase();
+    const socialPlatforms = ['facebook', 'twitter', 'github', 'youtube', 'linkedin', 'google'];
+    if (!socialPlatforms.includes(normalizedName)) {
```

Comment:
```suggestion
    if (this.name && !socialPlatforms.includes(normalizedName)) {
```

Reply:
No, the line I commented on will throw

Reply:
`normalizedName` will be equal to `globe` only if name exists.

Reply:
See the line where I commented. What'll happen if name is null?

Reply:
won't it return `this.name && this.name.trim().toLowerCase()` just like before ?

Reply:
You're looking at wrong line. Look at the line I commented on

Reply:
Oh, you mean the review comment. I was confused with the code comment.